contact-manager-integration-webhook-sync-extension/pom.xml

```diff
@@ -98,7 +87,7 @@
                         </goals>
                         <configuration>
```

Comment:
Shall we better use `generate-spring-boot-embedded` goal for this execution?

Reply:
I would keep it as it is for now, to close the PR and the task. But that's good hint to check 👍🏻